app/src/main/java/org/dhis2/data/forms/RulesRepository.kt

```diff
@@ -22,6 +19,9 @@ import org.hisp.dhis.rules.models.RuleEnrollment
 import org.hisp.dhis.rules.models.RuleEvent
 import org.hisp.dhis.rules.models.RuleVariable
 import timber.log.Timber
+import java.util.Calendar
+import java.util.Date
+import java.util.Objects
 
```

Comment:
Not in lexicographical order, does not comply with Kotlin code style

Reply:
ok...:(